patches/linker/0002-Update-linker-to-use-2.0.0-runtime.patch

```diff
@@ -0,0 +1,25 @@
+From c884d2eeacddbb3a09b8b2bac3c132ff3b3b63e4 Mon Sep 17 00:00:00 2001
+From: dseefeld <dseefeld@microsoft.com>
+Date: Mon, 15 Oct 2018 19:18:18 +0000
+Subject: [PATCH] Update linker to use 2.0.0 runtime
+
```

Comment:
Can you add something to the body about why this patch was added?

I am curious though if we could pass `/p:RuntimeFrameworkVersion=2.0.0` rather than using a patch for this, since IIRC properties set that way can't be changed... I don't mind the patch though.

Reply:
I'll update the patch.